specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/create.topics/create.topics/client.rpt

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2021-2023 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect "zilla://streams/app0"
+    option zilla:window 8192
+    option zilla:transmission "half-duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .request()
+                                 .createTopics()
+                                    .topic()
+                                       .name("test")
+                                       .numberOfPartitions(1)
```

Comment:
Do we need to be so verbose? Perhaps just `partitions` or `partitionsCount` is sufficient here?
Note that the name itself is not actually part of the protocol, just a label for the type of bytes in the protocol.

Reply:
actually you are right we can figure it out based on the assignments

Reply:
I wasn't suggesting we remove it. Actually, I don't think we can infer it from assignments since empty assignments implies automatic assignment.

I am suggesting we use a less verbose name for this, such as `partitionCount` or `partitions` instead of `numberOfPartitions`.
In the wire protocol on the website this is `num_partitions`.